ip_record.js

```diff
@@ -3,36 +3,89 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 // Keep track of events related to just IP addresses
-module.exports = function (BLOCK_INTERVAL_MS, now) {
+module.exports = function (BLOCK_INTERVAL_MS, now, RATE_LIMIT_INTERVAL_MS, MAX_ACCOUNT_STATUS_CHECK) {
```

Comment:
I believe `now` is intended to be an optional parameter, specified only when calling these from tests.  In which case we should leave it at the end of the parameter list and insert the new params before it.